boot/freeldr/freeldr/arch/realmode/amd64.S

```diff
@@ -29,15 +29,18 @@ Startup:
     mov byte ptr ds:[BSS_BootPartition], dh
 
     /* Setup a real mode stack */
-    mov sp, word ptr ds:[stack16]
+    mov sp, STACK16ADDR
```

Comment:
Just to be sure: is this change done only to reduce the opcode number of bytes, since we also know what the default value of the stack address is?